### PR TITLE
Add a minimum java version 

### PR DESCRIPTION
**Pull request by melissaanne**

### CODE REVIEW

**Comment by melissaanne** [comment]
Just a note, this isn't thoroughly tested on all boxed. I tested it out on centos 6 and debian 6, and it solves the issue Scott and I were seeing earlier when testing on centos 6.

**Comment by shrug** [comment]
:+1:

File: ext/debian/control.erb [hunk 1]
```diff
@@ -8,6 +8,6 @@ Homepage: http://www.puppetlabs.com
 
 Package: razor-torquebox
 Architecture: all
-Depends: ${misc:Depends}, default-jre-headless
+Depends: ${misc:Depends}, default-jre-headless (>= 1.6) | default-jre-headless (>= 1:1.6)
```
Comment (haus) [inline ext/debian/control.erb]: 1:1.6 is already greater than 1.6, so the second entry seems redundant.

Comment (melissaanne) [inline ext/debian/control.erb]: I agree, but the reason I kept this is that there are different available versions on different platforms. On one, only 1.6 is available, on another, 1:1.5 and 1:1.6 are both available. This was, in my head, the best way to deal with those differences. But I'm not positive, and totally open to help/suggestions

Comment (haus) [inline ext/debian/control.erb]: On lucid the default is 1.6 and on squeeze the default is 1:1.6. 1.5 hasn't been the default in debian or ubuntu for quite some time. I don't see 1.5 offered at all in deb or ubuntu.

Comment (melissaanne) [inline ext/debian/control.erb]: @haus updated

File: ext/redhat/razor-torquebox.spec.erb [hunk 2]
```diff
@@ -43,8 +43,8 @@ rm -rf %{buildroot}
 
 # Install the binary distribution.
 mkdir -p %{buildroot}/opt
-unzip -q -d %{buildroot}/opt torquebox-dist-%{version}-bin.zip
-mv %{buildroot}/opt/torquebox-%{version} %{buildroot}/opt/razor-torquebox
```
Comment (haus) [inline ext/redhat/razor-torquebox.spec.erb]: Why unmacro this and the next line?

Comment (melissaanne) [inline ext/redhat/razor-torquebox.spec.erb]: It's pulling the version number from the tag. I need it to correspond exactly to what the tarball is (3.1.1), whereas the version is evaluating to 3.1.1.2, which causes the build to fail. I could spend the time to actually make this work, but I'm lazy. I also explained this change in the commit message.

Comment (haus) [inline ext/redhat/razor-torquebox.spec.erb]: That would require reading and stuff, which I'm obviously incapable of.

**Comment by haus** [comment]
:+1:

**Pull request merged**